sources/osg/RenderStage.js

```diff
@@ -37,6 +30,41 @@ RenderStage.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit(
     // temporary, Utils.createPrototypeClass will solve this
     constructor: RenderStage,
 
+    cloneType: function () {
+        return this.constructor.prototype.getOrCreate().init();
```

Comment:
https://github.com/cedricpinson/osgjs/blob/develop/sources/osg/CullVisitor.js#L351
^^ changing that in a CullVisitor.getOrCreateRenderStage( type, args ) 
means replaceing that cullvisitor::_rootRenderStage mechanism by a cullvisitor override mechanism.